@ecomplus/storefront-app/src/components/html/CreditCardForm.html

```diff
@@ -3,10 +3,13 @@
   @submit.prevent="submit"
   novalidate
 >
-  <transition name="fade">
+  <transition
+    enter-active-class="animated fadeIn slow"
+    leave-active-class="animated fadeOut slow"
```

Comment:
```suggestion
    enter-active-class="animated fadeIn"
    leave-active-class="animated fadeOut"
```

não acho que o slow ficaria legal aqui na verdade, geralmente só quando o spinner sobrepõe a rota toda...